packages/blade/src/components/TopNav/_decisions/decisions.md

```diff
@@ -122,6 +122,157 @@ The container for the right-aligned actions like search, buttons, and user profi
 
 ### TabNav
 
+The horizontal navigation bar that contains multiple navigation items, each represented by TabNavItem. 
+
+TabNav's interaction is as follows: 
+
+1. There might already be items under More even at the highest screen size. These items might need a custom order.
+2. Once the screen size starts getting reduced, more items will start moving into More.
+3. The order of these items would follow a logical order. They keep getting added ahead of the already existing items. So the last item to go into More is the first item that you see when you click on More.
+
+TabNav automatically handles all these edge cases & responsiveness.
+
+```jsx
+<TabNav
```

Comment:
so on the top i saw API which says 
```
  <TopNavContent>
    <TabNav>
      <TabNavItem/>
    <TabNav/>
  <TopNavContent/>
  <TopNavAction>
     <Button/>
  <TopNavAction/>
```

so how does one differentiates between content and action because tab navigation is also an action but its inside the content sub-component

Reply:
Older API remnants. will change those.

Reply:
changed